fdbserver/workloads/IDDTxnProcessorApiCorrectness.actor.cpp

```diff
@@ -278,9 +353,24 @@ struct IDDTxnProcessorApiWorkload : TestWorkload {
 		params.startMoveKeysParallelismLock = &fl1;
 		params.finishMoveKeysParallelismLock = &fl2;
 		params.relocationIntervalId = relocateShardInterval.pairID;
+		TraceEvent(SevDebug, relocateShardInterval.begin(), relocateShardInterval.pairID)
+		    .detail("Key", params.keys)
+		    .detail("Dest", params.destinationTeam);
 
-		self->mock->moveKeys(params);
-		wait(self->real->moveKeys(params));
+		loop {
+			params.dataMovementComplete.reset();
+			wait(store(params.lock, self->real->takeMoveKeysLock(UID())));
+			try {
+				wait(self->mock->moveKeys(params));
+				wait(self->real->moveKeys(params));
+				break;
+			} catch (Error& e) {
+				if (e.code() != error_code_movekeys_conflict && e.code() != error_code_operation_failed)
+					throw;
+				wait(delay(FLOW_KNOBS->PREVENT_FAST_SPIN_DELAY));
```

Comment:
Usually we should not ignore `operation_failed` errors. What is the source of these errors here?